src/components/NewsList.tsx

```diff
@@ -105,23 +130,29 @@ export default function NewsList({ title, news, isLoading, to }: Props) {
           </div>
         </div>
         <div className="flex-1 hidden max-w-[300px] lg:flex flex-col gap-6">
-          {news?.slice(5, 8).map((article) => {
-            if (isLoading || typeof article === "number") {
+          {articles?.slice(5, 8).map((article) => {
+            if (typeof article === "number") {
               return (
-                <div key={article as number}>
+                <div key={article}>
                   <Skeleton className="w-full h-[120px] mb-2" />
                   <Skeleton className="w-full h-[43px] mb-2" />
                   <Skeleton className="w-full h-[20px]" />
                 </div>
               );
             }
```

Comment:
Use ternary or alternative operator here, instead of the very long if statement you're using.
You can do:
```jsx
{
    typeof article === "number" && <div></div>
}
```